api/src/Entity/MaterialItem.php

```diff
@@ -75,6 +77,10 @@ class MaterialItem extends BaseEntity implements BelongsToCampInterface, CopyFro
     /**
      * The name of the item that is required.
      */
+    #[InputFilter\Trim]
+    #[InputFilter\CleanText]
+    #[Assert\NotBlank]
+    #[Assert\Length(max: 32)]
```

Comment:
Isn't this is a bit too restrictive, to allow only 32 chars for the article name?

Reply:
What is your suggestion? 64, 128?
On mobile, 32 is a lot.
![image](https://user-images.githubusercontent.com/1506818/200128301-3b07523a-b3f1-4697-a0d0-c1d00101be16.png)
There we need a wrapping text input field that the user can edit the article.
On Desktop it is not a problem.
On PDF we can handle long texts.
It does not look nice on the pdf, but the user sees the information.

I would vote for 64, this was enough for my camps in ecamp2.

Reply:
64 sounds good to me.